File: Python/selectors/select_by_text.py
from playwright.sync_api import sync_playwright

def run():
    with sync_playwright() as p:
        browser = p.chromium.launch()
        page = browser.new_page()
        page.goto("https://example.com")
        element = page.get_by_text("Example Domain")
        print("By Text:", element.inner_text())
        browser.close()

run()

File: NodeJS/selectors/select_by_text.js
const { chromium } = require('playwright');

(async () => {
  const browser = await chromium.launch();
  const page = await browser.newPage();
  await page.goto('https://example.com');
  const element = await page.getByText('Example Domain');
  console.log('By Text:', await element.textContent());
  await browser.close();
})();